client/header/activity-panel/panels/inbox.js

```diff
@@ -33,6 +33,18 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	handleActionClick( event, note_id, action_id ) {
+		const { triggerNoteAction } = this.props;
+		const href = event.target.href || '';
+
+		if ( ! href.startsWith( wcSettings.adminUrl ) ) {
```

Comment:
We still need to verify `href` is not an empty string, right?

```suggestion
		if ( href.length && ! href.startsWith( wcSettings.adminUrl ) ) {
```